app/pathogen/arbovirus/dashboard/(map)/arbo-map-cluster-utils.tsx

```diff
@@ -0,0 +1,224 @@
+        </div>
+        <div style="display: flex; flex-direction: row; justify-content: space-between; padding-bottom: 0.5rem">
+            <div style="display: flex; flex-direction: column;">`;
+  if (props.properties.ZIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["ZIKV"]}">ZIKV</div>`;
+  if (props.properties.CHIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["CHIKV"]}">CHIKV</div>`;
+  if (props.properties.DENV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["DENV"]}">DENV</div>`;
+  if (props.properties.MAYV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["MAYV"]}">MAYV</div>`;
+  if (props.properties.YF > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["YF"]}">YF</div>`;
+  if (props.properties.WNV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem; padding-left: 5px; border-left: 5px solid ${pathogenColors["WNV"]}">WNV</div>`;
+  popupHTML += `</div>
+            <div style="display: flex; flex-direction: column;">`;
+
+  if (props.properties.ZIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.ZIKV}</div>`;
+  if (props.properties.CHIKV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.CHIKV}</div>`;
+  if (props.properties.DENV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.DENV}</div>`;
+  if (props.properties.MAYV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.MAYV}</div>`;
+  if (props.properties.YF > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.YF}</div>`;
+  if (props.properties.WNV > 0)
+    popupHTML += `<div style="font-size: 0.875rem; font-weight: 300; margin-bottom: 0.25rem;">${props.properties.WNV}</div>`;
+
+  popupHTML += `</div>
+        </div>
+        <div style="font-size: 0.875rem; font-weight: 300; text: center">Click to zoom in</div>
+
+    </div>`;
+
+  // Cannot use the react-gl Popup method as we need to use useRef to access the underlying method
+  // and we cannot do that when these popups are loaded dynamically and not statically.
+  const popup = new mapboxgl.Popup({
+    offset: [0, -7],
+    closeOnClick: false,
+  });
+
+  const onMouseEnter = (event: any) => {
```

Comment:
Again, could not implement this using the inbuilt react-map-gl stuff as this is not a layer and afaik there was no support for this in a dynamic setting.